### PR TITLE
Make certs valid starting yesterday

### PR DESCRIPTION
**Pull request by mberhault**

### CODE REVIEW

File: security/x509.go [hunk 1]
```diff
@@ -37,6 +37,8 @@ import (
 // Most fields and settings are hard-coded. TODO(marc): allow customization.
 
 const (
+	// Make certs valid a day before to handle clock issues (specifically boot2docker).
```
Comment (tamird) [inline security/x509.go]: please link to the specific issue https://github.com/boot2docker/boot2docker/issues/69

File: security/x509.go [hunk 2]
```diff
@@ -91,7 +93,7 @@ func newTemplate(commonName string) (*x509.Certificate, error) {
 			Organization: []string{"Cockroach"},
 			CommonName:   commonName,
 		},
-		NotBefore: notBefore,
+		NotBefore: notBefore.Add(validFrom),
```
Comment (tamird) [inline security/x509.go]: why not just change line 88?

Comment (MBerhault) [inline security/x509.go]: I liked the visual of having both next to each other. But I can if you really insist.

File: security/x509.go (hunk 1 above)
Comment (MBerhault) [inline security/x509.go]: Done.

File: security/x509.go (hunk 2 above)
Comment (tamird) [inline security/x509.go]: I don't feel that strongly, but it's weird to have this change span 50 lines

Comment (MBerhault) [inline security/x509.go]: ok. Done. Still spans 50 lines, but that's due to the constants.

**Pull request merged**